deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -1,5 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 bases:
 - ../../base
 patches:
-- controller_images.yaml
-- node_images.yaml
+- controller_add_snapshotter.yaml
+- controller_always_pull.yaml
+- node_always_pull.yaml
+images:
+- name: REPLACEME/gcp-compute-persistent-disk-csi-driver
+  # Replace this with your private image names and tags
+  newName: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver
+  newTag: "v0.4.0-gke.0"
```

Comment:
should this instead be not replaced instead. Therefore when someone tries to use a dev overlay without changing anything they get an image pull failure (saying `REPLACEME` and reminding the user to replace to image) instead of failing silently by just giving them a working image.

Reply:
We currently use the dev overlay as our alpha features overlay too (for snapshot).  I can create a separate overlay for alpha if we want to separate that out